teps/0048-task-results-without-results.md

```diff
@@ -631,6 +663,7 @@ spec:
       description: The pullRequestNumber
   tasks:
     - name: check-name-match
+      onError: continue
```

Comment:
is this valid? I don't see it in the api

Reply:
replied in comment https://github.com/tektoncd/community/pull/954#discussion_r1106640003

Reply:
this is not available yet at task level, only at step level.